examples/wrapper.js

```diff
@@ -1,23 +1,24 @@
-const ZooKeeper = require('../lib/index.js').Promise;
+const { constants, Promise: ZooKeeper } = require('../lib/index');
 
 const host = process.argv[2] || '127.0.0.1:2181';
 
 /**
  * @param timeoutMs {number}
- * @returns {ZooKeeperPromise}
+ * @returns {ZooKeeper}
```

Comment:
I think this one should be `ZooKeeperPromise` to get the correct client (promise based) intellisense in WebStorm and other IDEs.

Reply:
We import Promise as ZooKeeper so it should work. Otherwise we would have to import ZooKeeperPromise directly into this file.

Reply:
I know that WebStorm will parse the "class"/type (not the name of the imported module) that is specified in the jsDoc return statement. Even though the name is `ZooKeeper` it still is is the promise based class. I have tested it in WebStorm, but maybe other IDEs will manage it anyway?

Reply:
Okay, I've checked and it is working as you've described in IntelliJ. I have reverted this change.

Reply:
Great!